doc/source/conf.py

```diff
@@ -50,6 +50,7 @@
     'numpydoc',
     'IPython.sphinxext.ipython_directive',
     'IPython.sphinxext.ipython_console_highlighting',
+    'matplotlib.sphinxext.plot_directive',
 ]
 
 extlinks = {'issue': ('https://github.com/NSLS-II/{0}/issues/%s'.format(repo_name),
```

Comment:
```suggestion
extlinks = {'issue': ('https://github.com/bluesky/{0}/issues/%s'.format(repo_name),
```

Reply:
Not sure why the GH interface tries to touch the beginning of the line -- the line starts from the very beginning without blank spaces... Feel free to make the changes on your laptop instead.